management/WFCORE-1110_Custom_HTTP_Management_Headers.adoc

```diff
@@ -0,0 +1,84 @@
+= WFCORE-1110 Custom HTTP Headers for the HTTP Management Interface
+:author:            Darran Lofthouse
+:email:             darran.lofthouse@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-1110[WFCORE-1110]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-12787[WFLY-12787]
+* https://issues.jboss.org/browse/EAP7-1335[EAP7-1335]
+=== Dev Contacts
```

Comment:
Please put empty line above `=== Dev Contacts`. It isn't displaying correctly now.

Reply:
This comment is still valid.